_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,379 @@
+
+### Example: Requesting a PNG file
+
+The following command requests a report in PNG format with basic authentication and sends the report to an email address using Amazon SES:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:1234 -e ses -s <email address>  -r <email address> -f png
+```
+
+### Example: Requesting a PDF file
+
+The following command requests a PDF file and specifies the recipient's email address:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:Test@1234 -e ses -s <email address> -r <email address> -f pdf
+```
+
+Upon success, the file will be sent to the specified email address. The following image shows an example PDF report.
+
+![PDF example]({{site.url}}{{site.baseurl}}/images/dashboards/cli-pdf-report.png)
+
+### Example: Requesting a CSV file
+
+The following command generates a report that contains all table content in CSV format and sends the report to an email address using Amazon SES transport:
+
+```
+opensearch-reporting-cli -u https://search-basic-auth2-3-b477rrluhlckrmwmtvp6k3xxsi.us-west-2.es.amazonaws.com/_dashboards/goto/f166434919d2d8d3708a4705b6f2ed2d?security_tenant=global -f csv -a basic -c admin:Test1234 -e ses -s <email address> -r <email address>
+```
+
+Upon success, the email will be sent to the specified email address with the CSV file attached.
+
+## Creating a cron job to request a report
+
+You can create a report by setting up a cron job to run a report at a specified time.
+
+### Prerequisites
+
+You need a machine with Ubuntu and root access privileges.
+
+If cron is not installed by default, run the following command:
+
+```
+sudo apt install cron
+```
+
+### Step 1: Install the reporting CLI
+
+Install the Reporting CLI by running the following command:
+
+```
+npm i @opensearch-project/opensearch-reporting-cli
+```
+
+Open the crontab editor by running the following command:
+
+```
+crontab -e
+```
+In the crontab editor, enter the report request. For example, the following example shows a cron report that runs every day at 8am.
+
+```
+0 8 * * * opensearch-reporting-cli -u https://playground.opensearch.org/app/dashboards#/view/084aed50-6f48-11ed-a3d5-1ddbf0afc873 -e ses -s <sender_email> -r <recipient_email>
+```
+
+## Using AWS Lambda
+You can use AWS Lambda with the Reporting CLI tool. <need info on why they would use it>.
+
+### Prerequisites
+
+To use the Reporting CLI with AWS Lambda, you need to do the following preliminary steps.
+
+- Get an AWS account. For instructions, see [Creating an AWS account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-creating.html) in the AWS Account Management reference guide.
+- Get a docker file to assemble an image.
+
+You can use the following sample configurations to set up and assemble the image:
+
+```linux
```

Comment:
Use dockerfile instead of linux. Also please update dockerfile content with [updated] example. (https://github.com/opensearch-project/reporting-cli/blob/main/docs/examples/lambda_container_image.md)